v19.1/remove-nodes.md

```diff
@@ -127,25 +150,47 @@ No more data reported on target nodes. Please verify cluster health before remov
+### Step 4. Check the node and cluster after the decommissioning process
+
+In the Admin UI **Replication** dashboard, again hover over the **Replicas per Store** and **Leaseholders per Store** graphs. For the decommissioning node, the counts should be 0:
 
 <div style="text-align: center;"><img src="{{ 'images/v19.1/after-decommission1.png' | relative_url }}" alt="Decommission a single live node" style="border:1px solid #eee;max-width:100%" /></div>
 
 <div style="text-align: center;"><img src="{{ 'images/v19.1/after-decommission2.png' | relative_url }}" alt="Decommission a single live node" style="border:1px solid #eee;max-width:100%" /></div>
 
-Then view **Node List** on the **Overview** page and make sure all nodes but the one you removed are healthy (green):
+Then view **Node List** on the **Overview** page and make sure all nodes but the decommissioning node are healthy (green):
 
 <div style="text-align: center;"><img src="{{ 'images/v19.1/cluster-status-after-decommission1.png' | relative_url }}" alt="Decommission a single live node" style="border:1px solid #eee;max-width:100%" /></div>
 
+### Step 5. Stop the decommissioning node
+
+SSH to the machine running the node and execute the `cockroach quit` command:
+
+<div class="filter-content" markdown="1" data-scope="secure">
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach quit --certs-dir=certs --host=<address of decommissioned node>
+~~~
+</div>
+
+<div class="filter-content" markdown="1" data-scope="insecure">
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach quit --insecure --host=<address of decommissioned node>
+~~~
+</div>
+
 In about 5 minutes, you'll see the removed node listed under **Decommissioned Nodes**:
 
 <div style="text-align: center;"><img src="{{ 'images/v19.1/cluster-status-after-decommission2.png' | relative_url }}" alt="Decommission a single live node" style="border:1px solid #eee;max-width:100%" /></div>
 
-At this point, the node will no longer appear in timeseries graphs unless you are viewing a time range during which the node was live. However, it will never disappear from the **Decommissioned Nodes** list.
+At this point, the node is decommissioned and will no longer appear in timeseries graphs unless you are viewing a time range during which the node was live. However, it will never disappear from the **Decommissioned Nodes** list.
 
-Also, if the node is restarted, it will not accept any client connections, and the cluster will not rebalance any data to it; to make the cluster utilize the node again, you'd have to [recommission](#recommission-nodes) it.
+{{site.data.alerts.callout_info}}
+If you want to utilize a decommissioned node again, first [recommission](#recommission-nodes) the node to have the cluster rebalance data to the node. Then [restart](start-a-node.html) the node so that it accepts new SQL connections.
```

Comment:
In the future this will be impossible unless the data dir is wiped (i.e. it counts as a new node). May want to incorporate this in the docs already (it doesn't hurt to do this today already)

Reply:
@taroface, do you have a docs issue to follow-up here? Looks like a relatively quick fix we should make, if I'm understanding.

Reply:
Thanks! This is fixed in the v20.2 and v21.1 docs.

Reply:
Thanks! Also, I mentioned it in the public Slack, but it seems like a mistake that the removal of support for running `recommission` on a decommissioned node wasn't mentioned in the v20.2.0 release notes: https://www.cockroachlabs.com/docs/releases/v20.2.0

The commit/PR that made the change had a "general change" release note attached to it: https://github.com/cockroachdb/cockroach/pull/50329